pkgs/development/python-modules/sphinxcontrib-mscgen/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, sphinx
+, mscgen
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-mscgen";
+  version = "0.6";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-AXfSWRq3CepT/jNOgHxiYT7vkdKZejPu/LeUqxZ8T5A=";
+  };
+
+  propagatedBuildInputs = [
+    mscgen
+    sphinx
+  ];
+
+  # There are no unit tests
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "sphinxcontrib.mscgen"
+  ];
+
+  meta = with lib; {
+    description = "Sphinx extension using mscgen to render diagrams";
+    homepage = "https://github.com/sphinx-contrib/mscgen";
+    license = licenses.publicDomain;
```

Comment:
Wondering if we should introduce `BOLA` as a license, so others don't have to look it up.

Reply:
Done.